### PR TITLE
feat(app): create new ExitModal for Change pipette flow

### PR DESCRIPTION
**Pull request by jerader**
closes RLIQ-44

# Overview

Create new `ExitModal` and deprecate the `ExitAlertModal`

Screen Shot 2022-08-24 at 8 21 21 AM

# Changelog

- create `ExitModal` and a test
- deprecate `ExitAlertModal`
- small change to `children` prop and modal size in `SimpleWizardModal`
- fix deprecated tag info for the `ClearDeckAlertModal`
- remove `ClearDeckAlertModal/styles.css` because it is completely unused. I accidentally duplicated it in a previous PR (the duplicated `styles.css` is `ClearDeckModal/styles.css`

# Review requests

- go through the flow and click on the exit button. The modal title should be dynamic as well as the subHeader text depending on if you are attaching or detaching.
- go through the flow without the FF turned on, make sure it has retained the legacy styling

# Risk assessment

low

### CODE REVIEW

File: app/src/organisms/ChangePipette/ExitModal.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        iconColor={COLORS.warningEnabled}
+        header={t('progress_will_be_lost')}
+        subHeader={t('are_you_sure_exit', { direction: flow })}
+        onExit={exit}
+        isSuccess={false}
+        title={title}
+        currentStep={currentStep}
+        totalSteps={totalSteps}
+      >
+        <>
```
Comment (sakibh) [inline app/src/organisms/ChangePipette/ExitModal.tsx]: This fragment can be removed

File: app/src/molecules/SimpleWizardModal/index.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ export function SimpleWizardModal(props: Props): JSX.Element {
   } = props
 
   return (
-    <ModalShell width="47rem">
+    <ModalShell width="47rem" height="28.12rem">
```
Comment (sakibh) [inline app/src/molecules/SimpleWizardModal/index.tsx:40]: Enforcing this height adds a vertical scroll bar on the default viewport of the app. You can probably get around this quickly by adjusting the `marginTop` in `SimpleWizardModal` maybe from `6.8rem` to `5.8rem`.

image

**Review by sakibh — APPROVED** [review]
Looks great! Just left a couple small comments.

File: app/src/molecules/SimpleWizardModal/index.tsx (hunk 2 above)
Comment (jerader) [reply app/src/molecules/SimpleWizardModal/index.tsx:40]: ooof ya, good catch! I didn't notice the scroll.

File: app/src/organisms/ChangePipette/ExitModal.tsx (hunk 1 above)
Comment (jerader) [reply app/src/organisms/ChangePipette/ExitModal.tsx]: thanks! idk why i added that fragment 😆

**Pull request merged**